setup.py

```diff
@@ -60,8 +61,8 @@
     extras_require['all'].extend(reqs)
 
 setup_requires = [
-    'pytest-runner>=2.6.2',
     'Babel>=1.3',
```

Comment:
The package doesn't seem to have any translation strings so all the translation machinery can be removed including babel packages in setup, transfix config, and babel sections in setup.cfg